docs/overview.md

```diff
@@ -115,3 +115,23 @@ Software that is able to stand alone from users perspective. Groups more service
 #### Platform
 
 Even bigger whole that groups multiple applications (Console Dot Platform).
+
+---
+
+### Extended Terminology
+
+#### Advisory
+
+A formal notification from a vendor (e.g., Red Hat) regarding an issue in a software package that requires an update. Advisories provide structured information about the necessary update and are typically categorized by the type of problem they fix (e.g., security, bug fix, enhancement).
+
+#### EPEL & Community Repositories
+
+**EPEL** (Extra Packages for Enterprise Linux) repositories are read-only repositories sourced from the Fedora Project (e.g., `https://dl.fedoraproject.org/pub/epel/...`). Historically referred to as "Popular Repositories," they are now commonly designated as **Community Repositories** to reflect their automatic, shared availability across users and organizations. They are sometimes referred to as "unified EPEL" or "shared EPEL."
```

Comment:
```suggestion
**EPEL** (Extra Packages for Enterprise Linux) repositories are read-only repositories sourced from the Fedora Project (e.g., `https://dl.fedoraproject.org/pub/epel/...`). In the context of our app, EPEL repositories are available as **Community Repositories** to reflect their automatic, shared availability across users and organizations. 
```
One small comment. EPEL is usually just referred to as EPEL. These other names like "Popular Repositories" and "Community Repositories" are only used in the context of our app. And since unified EPEL is really just the name of the epic, I wouldn't document it.